gitpod-io/gitpod
# feat(gitpod-cli): report errors to ide-metrics-api

## Description
Introduce error reporting from `gp cli` (startin from `gp ports list`) to `ide-metrics-api`. Errors will be available in GCP.

## Related Issue(s)

Fixes #13707

## How to test
1. Start a workspace in prev env
2. Start a workspace in prod for this PR
3. Simulate an error in prod (e.g. take down server), expose a port, and run `gp ports list`
4. Observe the logs for `ide-metrics-api` using kubectl

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
